framework/include/utils/OffDiagonalScalingMatrix.h

```diff
@@ -0,0 +1,69 @@
+//* This file is part of the MOOSE framework
```

Comment:
I feel like this class better belong to libmesh

Reply:
Does @roystgnr think so? This idea of automatic scaling feels fairly specific to MOOSE right now. I don’t know if this class is generally useful?

Reply:
Might it be useful for an explicit  method to use a lumped mass matrix?

This is the right way to generate a lumped mass matrix

Reply:
That’s true

Reply:
Moved to libMesh. Now this file just does an alias